tgui/packages/tgui/interfaces/Secrets.js

```diff
@@ -0,0 +1,723 @@
+import { toFixed } from 'common/math';
+import { Fragment } from 'inferno';
+import { useBackend, useLocalState } from '../backend';
+import { Button, Flex, LabeledControls, NoticeBox, RoundGauge, Section } from '../components';
+import { FlexItem } from '../components/Flex';
+import { Window } from '../layouts';
+
+const formatPressure = value => {
+  return toFixed(value) + '%';
+};
+
+const lineHeightNormal = 2.79;
+const lineHeightDebug = 6;
+
+const DebuggingTab = (props, context) => {
+  const { act } = useBackend(context);
+  return (
+    <Flex
+      grow={1}
+      mb={-0.25}
+      mx={-0.5}
+      direction="column"
+      height="100%"
+      align="stretch"
+      justify="center">
+      <Flex.Item my={0.5}>
+        <Button
+          lineHeight={lineHeightDebug}
+          align="center"
+          icon="question"
+          fluid
+          content="Change all maintenance doors to engie/brig access only"
+          onClick={() => act("maint_access_engiebrig")} />
+      </Flex.Item>
+      <Flex.Item my={0.5}>
+        <Button
+          lineHeight={lineHeightDebug}
+          align="center"
+          icon="question"
+          fluid
+          content="Change all maintenance doors to brig access only"
+          onClick={() => act("maint_access_brig")} />
+      </Flex.Item>
+      <Flex.Item mt={0.5} mb={-0.5}>
+        <Button
+          lineHeight={lineHeightDebug}
+          align="center"
+          icon="question"
+          fluid
+          content="Remove cap on security officers"
+          onClick={() => act("infinite_sec")} />
+      </Flex.Item>
+    </Flex>
+  );
+};
+
+const HelpfulTab = (props, context) => {
+  const { act } = useBackend(context);
+  return (
+    <Flex direction="column" mb={-0.75} mx={-0.5}>
+      <Flex
```

Comment:
Have you tried just using wrapping properties of Flex? You would not need to manually split buttons in separate lines, and you would not need to use `direction="column"` as well, just one big flex.